### ISSUE TITLE
Enable Broken Podcast View

### ISSUE THREAD

**Issue by robertmain** [WP-for-Church/Sermon-Manager#208]
### Expected Behaviour

1. The podcast PHP view file should be displayed

### Actual Behaviour

1. Currently the custom actions are used to display the podcast feed

I'm trying to do some work on the podcast feed regarding the view file that is (according to the comment in there) "broken". I'd like to rewrite it using SimpleXML or DOMDocument (rather than just breaking in and out of strings all over the place). I think it would be much neater and easier to maintain.

The problem is - I cant' figure out how to get that view to display instead of the current PHP functions generating the podcast.

### Platform
**Sermon Manager Version:** 2.12.5
**WordPress Version:** 4.9.5
**PHP Version:** 7.9.1

**Comment by robertmain** [comment]
@nikola3244 is there anything I should know about translations? Anything (other than the scope) that I can't move into a class constant?

**Comment by nikola3244** [comment]
Sorry, I'm not exactly sure what is being asked, could you please re-phrase it?

Are we talking about WordPress translations? What does it exactly have to use with XML feed?
I don't think that anything should be translated in it, except maybe few small details

**Comment by robertmain** [comment]
Yeah, so remember I converted the scope parameter to a class constant and had to revert it because it broke translation? Well, is there anything else like that, that the translations depend on?

**Comment by nikola3244** [comment]
Oh right, translations text domain. And nah, there's nothing that I can think about right now

I'd like if translations stay the way they are, there's nothing wrong with them:
```
_e( 'Translate me!', 'sermon-manager-for-wordpress' );
```

**Comment by robertmain** [comment]
Yeah, I wasn't planning changing the translations. I just wanted to make sure I don't break them.

**Comment by nikola3244** [comment]
Great :)

Also, as you can see, I've closed this issue, since it's done (sorry, I know you wanted to do it :pray:). 
I found some of my old code and realized how close it is to being done. Changed few lines, and voila, it works. After that, I've just added all the XML tags and data

I think it all works correctly, but it needs testing

**Comment by robertmain** [comment]
No, is all good. I don't mind you closing it :)
